src/semantic-ui/site/globals/site.overrides

```diff
@@ -142,7 +142,9 @@ a {
     margin-left: 1.6em;
   }
   &.frontsite-mobile {
+    position: absolute;
```

Comment:
what is this solving? I am not a fan of negative margins! :)

Reply:
It's only affecting the `environmentLabel` and it basically prevents it from interfiering with other components/tags (since it's not part of the production environment)
![image](https://user-images.githubusercontent.com/15194802/109177956-a3a9bd80-7788-11eb-9d77-845134e8e440.png)

Reply:
maybe only for mobile we could display it like this, over the site title element (CERN Library Catalogue) instead of going with specific css (just an idea, if it is not worth the time, don't change it ;) )
<img width="370" alt="Screenshot 2021-02-26 at 14 36 57" src="https://user-images.githubusercontent.com/38131488/109306870-175ad180-7840-11eb-9026-d80fef047dcd.png">

Reply:
We would still need specific css, since it gets cut on top:
![image](https://user-images.githubusercontent.com/15194802/109311273-e67d9b00-7845-11eb-987b-a337af679c28.png)